posts/2021-08-20-open-liberty-starter.adoc

```diff
@@ -33,7 +33,7 @@ image::img/blog/starter_application.png[screen capture of the starter applicatio
 
 Once you download the starter project, unpackage the .zip file on your machine.
 
-If you selected `Maven` as your build tool, then open a command line session, navigate to the installation directory, and run `mvnw liberty:run`. This will install all required dependencies and start the default server. If successful, you will see the necessary features installed and the message "server is ready to run a smarter planet."
+If you selected `Maven` as your build tool, then open a command line session, navigate to the installation directory, and run `mvn liberty:run`. This will install all required dependencies and start the default server. If successful, you will see the necessary features installed and the message "server is ready to run a smarter planet."
```

Comment:
I'm not sure why this is getting pulled in, but it's incorrect; it should be mvnw (it use to be mvn, and was fixed to be mvnw)..... perhaps you created this post off of `draft` branch and the changed wasn't properly backported to it? Always create your post off of `prod` branch, never off of `draft`